cv.tex

```diff
@@ -44,92 +50,85 @@
 % Education
 %------------------------------------------------
 
-\section{Education} 
-
-\subsection{Cornell University}
-
-\descript{MEng in Computer Science}
-\location{Expected Dec 2014 | Ithaca, NY \\ Cum. GPA: N/A}
+\section{Education}
+\subsection{\href{https://www.ucf.edu/}{UCF}}
+\descript{\href{https://www.ucf.edu/degree/information-technology-bs/}{B.S. in Information Technology}}
+\location{December 2014 | Orlando, Fl}
+\href{https://www.cecs.ucf.edu/}{College of Eng. \& Computer Sciences} \\
+Dean's List (Later Semesters) \\
+\location{Major GPA: 3.44 / 4.0}
 
 \sectionspace % Some whitespace after the section
 
-\descript{BS in Computer Science}
-\location{Expected May 2014 | Ithaca, NY}
-Conc. in Software Engineering \\
-College of Engineering \\
-Dean's List (All Semesters) \\
-\location{ Cum. GPA: 3.92 / 4.0 \\
-Major GPA: 3.94 / 4.0}
-
-\sectionspace % Some whitespace after the section
 
+%------------------------------------------------
+% Certifications
 %------------------------------------------------
 
-\subsection{La Martiniere for Boys}
-
-\location{Grad. May 2011 | Kolkata, India}
-
+\section{Certifications}
+\subsection{\href{https://www.giac.org/}{GIAC} (Expired)}
+2017 \href{https://www.giac.org/certifications/cyber-threat-intelligence-gcti/}{GCTI} (\#1144) \\
+2017 \href{https://www.giac.org/certifications/certified-forensic-analyst-gcfa/}{GCFA} (\#13368) \\
+2019 \href{https://www.giac.org/certifications/web-application-penetration-tester-gwapt/}{GWAPT} (\#6137) \\
 \sectionspace % Some whitespace after the section
 
 %------------------------------------------------
-% Links
+% Skills
 %------------------------------------------------
 
-\section{Links} 
+\section{Skills}
 
-Github:// \href{https://github.com/deedydas}{\bf deedydas} \\
-LinkedIn:// \href{https://www.linkedin.com/in/debarghyadas}{\bf debarghyadas} \\
-YouTube:// \href{https://www.youtube.com/user/DeedyDash007}{\bf DeedyDash007} \\
-Twitter:// \href{https://twitter.com/debarghya_das}{\bf @debarghya\_das} \\
-Quora:// \href{https://www.quora.com/Debarghya-Das}{\bf Debarghya-Das}
+\subsection{Programming}
```

Comment:
(JC):
Under Programming, I see some things listed and then two sections: Working Pro
and Familiar. Those make sense but then what are the items like AWS and Python? (Same feedback for Analytical.

What about creating a another section so you have these: Programming, Cloud, and Analytic? Clouds aren’t really programming. If you did this, I just listed them in order of familiarity rather than having Working Proficiency and Familiar.

Reply:
(JC):
- You have Splunk listed in Analytic al. Should you list SPL under Programming (kind of like SQL)?
- Oof. OCI? (BTW, had to look that up.) I can only imagine what that’s like.
- Are JSON and YAML worth listing? Data formats. I don’t think they add value.
- “Javascript” —> “JavaScript”

Reply:
reformatted the section, agree it makes more sense this way but in print it looks wonky. further work is needed to indent or otherwise modify content to get something easier on the eye, accepted all suggestions